src/Components/LetterBuilder/ProgressBar/ProgressBar.tsx

```diff
@@ -20,10 +20,14 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
     <div className="progress-bar">
       <div id="progress-bar-title" className="progress-bar__title">
         <h2>
-          <Trans>Build your letter</Trans>
+          <Trans>
+            <strong>Letter Builder</strong>
+          </Trans>
```

Comment:
Just to keep the lingui strings cleaner - avoiding `<0>Letter Builder</0>`
```suggestion
          <strong>
           <Trans> Letter Builder</Trans>
          </strong>
```